src/widgetastic_patternfly/__init__.py

```diff
@@ -36,6 +36,12 @@
 
 from .utils import PFIcon
 
+# py3.7 support
```

Comment:
What is this intending to fix? This newly named `Pattern` isn't used anywhere in the module - and there are a few direct references to `re._pattern_type` here.

Reply:
I missed... let me correct them.